src/configure/utilities/utilities.ts

```diff
@@ -0,0 +1,21 @@
+import * as crypto from 'crypto';
+
+export class Utilities {
+    public static createSha256Hash(input: string): string {
+        return crypto.createHash('sha256').update(input).digest('hex');
+    }
+}
+
+export class WhiteListedError extends Error {
+    constructor(message?: string, error?: any) {
```

Comment:
do we really need to clone the error? Will this work?
```
export class WhiteListedError extends Error {
    constructor(message?: string) {
        super(message);
    }
}
```
And we can rely on type casting to convert from `Error` to `WhiteListedError`

Reply:
I want to preserve the actual stacktrace, thus I am cloning the error.